docs/en/seatunnel-engine/logging.md

```diff
@@ -80,6 +80,28 @@ appender.file.layout.pattern = [%X{ST-JID}] %d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%-
 
 SeaTunnel Engine automatically integrates Log framework bridge, allowing existing applications that work against Log4j1/Logback classes to continue working.
 
+### SeaTunnel Log Configuration
```

Comment:
why not we binding log clear with job ttl in JobHistoryService? https://github.com/apache/seatunnel/blob/cd9836bced8cf9ccddc72f68fbee051f201b9678/seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/master/JobHistoryService.java#L111
cc @dailai @hailin0

Reply:
This is a good suggestion.

Reply:
get